docs/source/api/user-guide/transformations/aggregation-quantile.ipynb

```diff
@@ -1,374 +1,515 @@
+                        "text/plain": [
+                            "<Figure size 640x480 with 1 Axes>"
+                        ]
+                    },
+                    "metadata": {},
+                    "output_type": "display_data"
+                }
+            ],
+            "source": [
+                "def sample_error(meas):\n",
+                "    return np.linalg.norm(true_quantiles - meas(data))\n",
+                "\n",
+                "m_expo_quantiles = make_expo_quantiles(quartiles, max_contributions, epsilon)\n",
+                "expo_err = np.mean([sample_error(m_expo_quantiles) for _ in range(25)])\n",
```

Comment:
Maybe move this inside the function, so all the `*_err` are parallel?

Reply:
I kept it out because it runs slower, but I updated the text to make it clearer.